build/bin/build-server-rpm

```diff
@@ -231,6 +231,17 @@ build() {
                 cp -r /home/builder/clickhouse-rpm/rpmbuild/RPMS/* /home/builder/rpm/RPMS/${spec_name}-${rpm_version}
                 ls -la /home/builder/clickhouse-rpm/rpmbuild/SOURCES
                 cp -r /home/builder/clickhouse-rpm/rpmbuild/SOURCES/* /home/builder/rpm/SOURCES
+            elif [ \"${spec_name}\" == \"dbaas-tools\" ]; then
```

Comment:
Please do not add more there – we want to remove that hack, not add to it.

I think we can add GoReleaser installation to the spec itself? Or do something else, but do not add there.

Reply:
GoReleaser is not available in repos(https://pkgs.org/search/?q=goreleaser), I had to install it by their way(documentation says to download the package from their GH release page):

```
deb/rpm:

Download the .deb or .rpm from the releases page and install with dpkg -i and rpm -i respectively.
```
https://goreleaser.com/install/

Reply:
No, `go get` is not better, do not use `go get`.

What I'm saying is that we should not do any that in that huge `if` statement hack that was added for ClickHouse and that we want to remove.